### DISCUSSION TITLE
Duet?

### DISCUSSION THREAD

**Discussion by eecue** [Jyers/Marlin#701] [Q&A] [answered]
If I swap out my main board for a Duet can I still use your firmware @Jyers ? I’m happy to tweak and compile as needed. I’d like to have separate control of my dual Z steppers so I can use the Z alignment Marlin command.

**Comment by Jyers** [comment, marked as answer]
As long as you are able to connect the V2 display you should be golden.

**Comment by eecue** [comment]
Oh nice. Guessing that goes for any board?

**Reply by Jyers** [reply]
I would assume so, as long as there is a way to connect the display any board should hypothetically work.
